lib/reporting/command_line_options.rb

```diff
@@ -40,21 +50,42 @@ def parse!(argv, out: STDOUT)
           exit 1
         end
       end
+      # rubocop:enable Metrics/BlockLength
 
       parser.parse!(argv)
 
       if !date || !issuer
         out.puts parser
         exit 1
+      else
+        {
+          time_range: time_range(date:, period:),
+          issuer: issuer,
+          verbose: verbose,
+          progress: progress,
+        }
       end
-
-      {
-        date: date,
-        issuer: issuer,
-        verbose: verbose,
-        progress: progress,
-      }
     end
     # rubocop:enable Rails/Exit
+
+    # @param [Date]
+    # @return [Range<Time>]
+    def time_range(date:, period:)
+      orig_beginning_of_week = Date.beginning_of_week
+      Date.beginning_of_week = :sunday
```

Comment:
[question] is monday the default beginning of week for `Date?`

Reply:
yep

Reply:
Also we need to get clarification from the partner when they want these reports to start. It sounded [in Slack](https://gsa-tts.slack.com/archives/C01SU3NB9T3/p1679488393051929?thread_ts=1679420203.210389&cid=C01SU3NB9T3) like maybe if they want the report each Monday, that the weel should *end* on Sunday? (start on Monday?)

Reply:
I'm going to merge with the Sunday-Saturday week and we can always follow up with another PR if needed to adjust that